<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>迷宮探索</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="MazeCreateGrop">
        <property name="title">
         <string>迷宮基設定區域</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>長度 (M)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="mazeM_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>寬度(N)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="mazeN_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="mazeSizeConfirm_pushButton">
           <property name="text">
            <string>確認設定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="PaintingGroup">
        <property name="title">
         <string>起始點  / 終點設定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="2">
            <widget class="QSpinBox" name="mazeStartPoint_Y_spinBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>終點設定</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="mazeStartPoint_X_spinBox">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="mazeEndPoint_Y_spinBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>起始點</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="mazeEndPoint_X_spinBox"/>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="chooseStartPoint_pushButton">
             <property name="text">
              <string>選取起始點</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="chooseEndPoint_pushButton">
             <property name="text">
              <string> 選取終點</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="mazePaintConfirm_pushButton">
           <property name="text">
            <string>繪製確認 / 尋找解答</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MazeQtGraphicsView" name="mazeView_graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>說明</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>檔案</string>
    </property>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MazeQtGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">lib/Maze/Qt/MazeQtGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
